semantic_release_template.yaml

```diff
@@ -0,0 +1,12 @@
+name: screwdriver-cd/semantic-release
```

Comment:
since we don't allow any template inheritance model, how does this work in real world ? To do semantic release will users be forced to run a separate job whose only purpose is publish ? But then this new job will have to do build step as well.

Reply:
We discussed offline. The majority of the NPM modules have a dedicated `publish` job.

The ones that do not use `semantic-release` are intentionally configured that way.